src/resources/bitcoin/transaction.ts

```diff
@@ -105,20 +104,12 @@ export function getOpenTxnLink(params: {
 
 export async function getTxnHex(params: { hash: string; coinType: string }) {
   const res = await http
```

Comment:
We can just return the response like other functions.

Reply:
okay (y)